notebooks/1.0-vse-eda-rooms.ipynb

```diff
@@ -0,0 +1,614 @@
+{
```

Comment:
<ul><li>First of all, congratulation on your first PR! Santa Maria rules</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>